modules/removing-bare-metal-hosts-from-provisioner.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+:_content-type: PROCEDURE
+[id="removing-bare-metal-hosts-from-provisioner_{context}"]
+= Removing bare metal hosts from the provisioner node
+
+In certain circumstances it is desirable to temporarily remove bare metal hosts from the provisioner node.
+For example, during provisioning when a bare metal host reboot is triggered by using the {product-title} administration console or as a result of a Machine Config Pool update, {product-title} logs into the integrated Dell Remote Access Controller (iDrac) and issues a delete of the job queue.
+
+To prevent management of the number of `Machine` objects that matches the number of available `BareMetalHost` objects, add a `baremetalhost.metal3.io/detached` annotation to the `MachineSet` object.
+[NOTE]
+====
+This annotation only has an effect for `BareMetalHost` objects that are in either `Provisioned`, `ExternallyProvisioned` or `Ready/Available` state.
+====
+
+.Prerequisites
+
+* Install {op-system} bare metal compute machines for use in the cluster, and create corresponding `BareMetalHost` objects.
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Annotate the compute machine set that you want to remove from the provisioner node by adding the `baremetalhost.metal3.io/detached` annotation. Replace `<machineset>` with the name of the compute machine set.
++
+[source,terminal]
+----
+$ oc annotate machineset <machineset> -n openshift-machine-api 'baremetalhost.metal3.io/detached'
+----
++
+Wait for the new machines to start.
++
+[NOTE]
+====
+When you use a `BareMetalHost` object to create a machine in the cluster and labels or selectors are subsequently changed on the `BareMetalHost`, the `BareMetalHost` object continues be counted against the `MachineSet` that the `Machine` object was created from.
```

Comment:
If you used a `BareMetalHost` object to create a machine in the cluster, and 
(I know you copied this one, too; but, I don't think it's correct.)

Reply:
I feel like leaving this in as I think if throws up a question in my mind